src/calendar-app/calendar-app.ts

```diff
@@ -210,7 +210,7 @@ import("../mail-app/translations/en.js")
 			calendar: makeViewResolver<
 				CalendarViewAttrs,
 				CalendarView,
-				{ drawerAttrsFactory: () => DrawerMenuAttrs; header: AppHeaderAttrs; calendarViewModel: CalendarViewModel }
+				{ drawerAttrsFactory: () => DrawerMenuAttrs; header: AppHeaderAttrs; calendarViewModel: CalendarViewModel; bottomNav: Children }
```

Comment:
this will cache vnodes which is usually a bad idea. We should either make it a function or attributes